### DISCUSSION TITLE
Cannot see Example Adapter

### DISCUSSION THREAD

**Discussion by dairoca90** [openTCS/opentcs#10] [Q&A] [answered]
After i download the integration example i wanted to see the example communication adapter in the kernel control center but its only  showing the loopback adapter how can i see the adapter?

**Comment by martingr** [comment, marked as answer]
In order to make the example communication adapter available in the Kernel Control Center application you have to modify the plant model in the Model Editor application and add the `example:vehicleHost` and `example:vehiclePort` properties to the vehicle elements. These properties are also described in the openTCS integration project example's [documentation](https://github.com/openTCS/opentcs-integration-example/blob/master/openTCS-Example-Documentation/src/docs/asciidoc/opentcs-example-commadapter.adoc).

**Reply by dairoca90** [reply]
Thanks, i was reading only the developers guide, i didnt look at the documentation example now its working 👍
